### PR TITLE
EDUCATOR-5185: Team Final Grade not appearing in "Manage Team Responses" for staff graded Team ORA

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** Fix a bug where a Team's Final Grade sometimes doesn't show up in the staff area even when the staff assessment has been done

**What changed?**

- added function `get_team_submission_uuid_from_individual_submission_uuid` to `team_mixin` which basically just calls `get_team_submission_from_individual_submission` in the team submission api
- Previously, in `get_workflow_info`, we checked if the ORA was a team ORA and if it was, we called `get_team_workflow info` with no arguments. That works in the case where it's a student loading the ORA, but when it's a staff member, they can't "derive" the correct `team_submission_id`.
- Now, if `get_workflow_info` is called with a `submission_uuid` (as it is when the staff area is loading the workflow) and it's a team ORA, we take the `submission_uuid` and look up the associated `team_submission_uuid` and pass that to `get_team_workflow info`, which allows us to correctly look up the workflow.

**Developer Checklist**
- [x] Reviewed the [release process](./release_process.md)
- [x] Translations up to date
- [x] JS minified, SASS compiled
- [x] Test suites passing on Jenkins
- [x] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [EDUCATOR-5185](https://openedx.atlassian.net/browse/EDUCATOR-5185)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by muselesscreator** [comment]
Missing description. Is this just updating tests as part of another fix?

**Comment by jansenk** [comment]
This isn't ready for review, I just wanted to see if the tests passed

File: openassessment/xblock/test/test_staff_area.py [hunk 1]
```diff
@@ -714,6 +715,38 @@ def test_cancel_team_submission(self, xblock):
...
+        """
+        # Set up team assignment and submission. StaffMember is not on any team.
+        team_submission = self._setup_xblock_and_create_submission(
+            xblock,
+            anonymous_user_id='StaffMember',
+            team=True,
+            has_team=False
+        )
+        # Assess team submission
+        feedback = "Team assignment complete!!!!!"
```
Comment (Andytr1) [inline openassessment/xblock/test/test_staff_area.py:731]: I like the extra `!` :)
